testsuite/MDAnalysisTests/coordinates/test_pdb.py

```diff
@@ -636,6 +636,8 @@ def test_iteration(self, multiverse):
             "iterated number of frames %d is not the expected number %d; "
             "trajectory iterator fails to rewind" %
             (len(frames), u.trajectory.n_frames))
+        assert_equal(u.trajectory.ts.frame, 0,
```

Comment:
I think this is a good idea both here and in `test_xdr` and I am happy to leave them in but as we are dealing with behaviour from the `ProtoReader` base class, I think we will also need a test of the base class behaviour. 

 I would love to see a test in `testsuite/MDAnalysisTests/coordinates/base.py` called something like `test_iteration_rewinds` where you compare where the trajectory ends up from the different types of iteration. 

 Other people feel free to weigh in also.  :)

Reply:
@hmacdope I added a `test_iteration_rewinds` to `testsuite/MDAnalysisTests/coordinates/base.py`. However, I'm not entirely sure it's what you were intending. Any feedback is appreciated! :) Thank you for your help!

Reply:
Yep looks spot on to me!